test/controllers/api/docs_controller_test.exs

```diff
@@ -32,83 +27,62 @@ defmodule HexWeb.API.DocsControllerTest do
 
     conn = get build_conn(), "docs/phoenix/index.html"
     assert conn.status == 200
-    assert conn.resp_body == "HEYO"
+    assert conn.resp_body == "phoenix v0.0.1"
```

Comment:
I initially thought about using `html_response` but I got an error that Content-Type isn't set (I guess we were missing that in the test controller) and gave up for now. I did not know about the `response(conn, xxx)` helper and it's perfect, I'll use it 😅 I'll also fix the Content-Type thingy later